src/Akka.Persistence.MongoDb.Hosting/AkkaPersistenceMongoDbHostingExtensions.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
+using Akka.Actor;
+using Akka.Hosting;
+using Akka.Persistence.Hosting;
+
+#nullable enable
+namespace Akka.Persistence.MongoDb.Hosting;
+
+public static class AkkaPersistenceMongoDbHostingExtensions
+{
+    /// <summary>
+    ///     Adds Akka.Persistence.SqlServer support to this <see cref="ActorSystem"/>.
+    /// </summary>
+    /// <param name="builder">
+    ///     The builder instance being configured.
+    /// </param>
+    /// <param name="connectionString">
+    ///     Connection string used for database access.
+    /// </param>
+    /// <param name="mode">
+    ///     <para>
+    ///         Determines which settings should be added by this method call.
+    ///     </para>
+    ///     <i>Default</i>: <see cref="PersistenceMode.Both"/>
+    /// </param>
+    /// <param name="autoInitialize">
+    ///     <para>
+    ///         Should the SQL store table be initialized automatically.
+    ///     </para>
+    ///     <i>Default</i>: <c>false</c>
+    /// </param>
+    /// <param name="journalBuilder">
+    ///     <para>
+    ///         An <see cref="Action{T}"/> used to configure an <see cref="AkkaPersistenceJournalBuilder"/> instance.
+    ///     </para>
+    ///     <i>Default</i>: <c>null</c>
+    /// </param>
+    /// <param name="pluginIdentifier">
+    ///     <para>
+    ///         The configuration identifier for the plugins
+    ///     </para>
+    ///     <i>Default</i>: <c>"sql-server"</c>
+    /// </param>
+    /// <param name="isDefaultPlugin">
+    ///     <para>
+    ///         A <c>bool</c> flag to set the plugin as the default persistence plugin for the <see cref="ActorSystem"/>
+    ///     </para>
+    ///     <b>Default</b>: <c>true</c>
+    /// </param>
+    /// <returns>
+    ///     The same <see cref="AkkaConfigurationBuilder"/> instance originally passed in.
+    /// </returns>
+    /// <exception cref="ArgumentOutOfRangeException">
+    ///     Thrown when <see cref="journalBuilder"/> is set and <see cref="mode"/> is set to
+    ///     <see cref="PersistenceMode.SnapshotStore"/>
+    /// </exception>
+    public static AkkaConfigurationBuilder WithMongoDbPersistence(
+        this AkkaConfigurationBuilder builder,
+        string connectionString,
```

Comment:
In this method I have only added the `connectionString` as a parameter that is custom on top of the default. For this plugin specifically are there any others that you would suggest adding?